packages/frontend/src/features/common/components/layout/userPanel/UserSection.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { FontSize } from "../../../styles/constans/FontSize"
+
+import UserData from "./UserData"
+
+const Wrapper = styled.div`
+    padding-bottom: 10px;
```

Comment:
a co z padding top i cieniem?

Reply:
cień  i padding jest na całym wrapperze to jest tylko sekcja od zdjęcia do przycisku wyloguj

Reply:
wycofuje uwagę